cmd/main.go

```diff
@@ -851,3 +856,33 @@ func createVersionHandlers(log logrus.FieldLogger, ctrlMgr manager.Manager, rele
 
 	return versionsHandler, versionsAPIHandler, nil
 }
+
+func getKafkaWriter(log *logrus.Logger) (*stream.KafkaWriter, error) {
+
+	metadata := map[string]interface{}{
+		"versions": Options.Versions,
+	}
+	eventStreamWriter, err := stream.NewKafkaWriterWithMetadata(metadata)
+	if err != nil {
+		return nil, errors.New("error instantiating kafka client")
```

Comment:
I think we should wrap the original error to make sure we don't lose this information